usagov/usagov-2021
# usagov-2120-page-region-attribute: update logic to remove role attrib…

…ute if no image is chosen


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2120

## Description

Update code logic to remove role=“region” and area-label for alert box with no image or title
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Goto http://localhost/voter-registration-card and inspect the info alert box to verify that the code contains role="region"
and aria-label="info". This alert box should contain an image and a title.
<img width="1512" alt="Screenshot 2024-12-13 at 9 35 18 AM" src="https://github.com/user-attachments/assets/eedd59be-b7eb-41e0-bb18-0f1f77f5617c" />

- Goto https://www.usa.gov/adjustment-of-status and inspect the alert box with the gray left bar in the content area to verify that the code does not contain role="region" and aria-label="info". This alert box should contain no image or title.
<img width="1512" alt="Screenshot 2024-12-13 at 9 34 48 AM" src="https://github.com/user-attachments/assets/3e48e116-a07c-41f2-9492-bd6a87eedd2e" />

- Verify the same steps on the Spanish pages below.
- http://localhost/es/solicitar-tarjeta-registro-electoral
<img width="1512" alt="Screenshot 2024-12-13 at 9 35 57 AM" src="https://github.com/user-attachments/assets/874858db-1113-4548-b81e-d8b5bb05d201" />

- http://localhost/es/ano-nuevo
<img width="1512" alt="Screenshot 2024-12-13 at 9 37 07 AM" src="https://github.com/user-attachments/assets/d1cf704e-517f-4190-a353-0f86a1912f59" />

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
